src/workerd/api/http.c++

```diff
@@ -1122,7 +1134,7 @@ jsg::Ref<Request> Request::clone(jsg::Lock& js) {
   auto bodyClone = Body::clone(js);
 
   return jsg::alloc<Request>(method, url, redirect, kj::mv(headersClone), getFetcher(), getSignal(),
-      kj::mv(cfClone), kj::mv(bodyClone));
+      kj::mv(cfClone), kj::mv(bodyClone), cacheMode, responseBodyEncoding);
```

Comment:
I _think_ the absence of `cacheMode` here was a bug? The effect would be that `.clone()` on a request would not copy over the `cacheMode` correctly.